deps/rabbitmq_cli/lib/rabbitmq/cli/core/ansi.ex

```diff
@@ -6,30 +6,37 @@
 
 defmodule RabbitMQ.CLI.Core.ANSI do
   def bright(string) do
-    "#{IO.ANSI.bright()}#{string}#{IO.ANSI.reset()}"
+    colorize(%{string: string, code: "#{IO.ANSI.bright()}"})
   end
 
   def red(string) do
-    "#{IO.ANSI.red()}#{string}#{IO.ANSI.reset()}"
+    colorize(%{string: string, code: "#{IO.ANSI.red()}"})
   end
 
   def yellow(string) do
-    "#{IO.ANSI.yellow()}#{string}#{IO.ANSI.reset()}"
+    colorize(%{string: string, code: "#{IO.ANSI.yellow()}"})
   end
 
   def magenta(string) do
-    "#{IO.ANSI.magenta()}#{string}#{IO.ANSI.reset()}"
+    colorize(%{string: string, code: "#{IO.ANSI.magenta()}"})
   end
 
   def bright_red(string) do
-    "#{IO.ANSI.bright()}#{IO.ANSI.red()}#{string}#{IO.ANSI.reset()}"
+    colorize(%{string: string, code: "#{IO.ANSI.bright()}#{IO.ANSI.red()}"})
   end
 
   def bright_yellow(string) do
-    "#{IO.ANSI.bright()}#{IO.ANSI.yellow()}#{string}#{IO.ANSI.reset()}"
+    colorize(%{string: string, code: "#{IO.ANSI.bright()}#{IO.ANSI.yellow()}"})
   end
 
   def bright_magenta(string) do
-    "#{IO.ANSI.bright()}#{IO.ANSI.magenta()}#{string}#{IO.ANSI.reset()}"
+    colorize(%{string: string, code: "#{IO.ANSI.bright()}#{IO.ANSI.magenta()}"})
+  end
+
+  defp colorize(%{string: string, code: color}) do
```

Comment:
This function should probably be named `maybe_colorize/2` since it does not always have an effect.

Reply:
Done, @michaelklishin; thank you.